content/en-us/production/promotion/invite-prompts.md

```diff
@@ -118,13 +118,10 @@ By default, the invite notification that friends receive contains the sender's `
 1. In the center region, click the **Create a Notification String** button.
 1. Fill in an identifier name (only visible to you) and the custom notification text. Note that you must include **\{experienceName\}** as a placeholder to identify the experience's name for invited friends, and you can optionally include the sender's `Class.Player.DisplayName|DisplayName` through the **\{displayName\}** placeholder.
 
-   <Alert severity="success">
-   <AlertTitle>Example Notifications</AlertTitle>
-	 <ul>
-	 <li>**\{displayName\}** wants you to join their adventure in **\{experienceName\}**!</li>
-	 <li>**\{displayName\}** just cleared the sixth stage of **\{experienceName\}**. Can you?</li>
-	 </ul>
-	 </Alert>
+   Example notification strings:
```

Comment:
Opting to just take these out of the Alert box and make them bullets with styling, similar to https://create.roblox.com/docs/production/promotion/experience-notifications#creating-a-notification-string